gophish_cloud_init.tf

```diff
@@ -37,9 +37,55 @@ data "cloudinit_config" "gophish_cloud_init_tasks" {
     })
     content_type = "text/x-shellscript"
     filename     = "mount-efs-share.sh"
+  }
+
+  # Create the JSON file used to configure Docker daemon.  This allows us
+  # to tell Docker to store volume data on our persistent
+  # EBS Docker data volume (created below).
+  part {
+    content = templatefile(
+      "${path.module}/cloud-init/write-docker-daemon-json.tpl.yml", {
+        docker_data_root_dir = local.docker_volume_mount_point
+    })
+    content_type = "text/cloud-config"
+    filename     = "write-docker-daemon-json.yml"
     merge_type   = "list(append)+dict(recurse_array)+str()"
   }
 
+  # Prepare and mount EBS volume to hold Docker data-root data.
+  # Note that this script and the next one must take place in a certain order,
+  # so we prepend numbers to the script names to force that to that happen.
+  #
+  # Here is where the user scripts are called by cloud-init:
+  # https://github.com/canonical/cloud-init/blob/master/cloudinit/config/cc_scripts_user.py#L45
+  #
+  # And here is where you can see how cloud-init sorts the scripts:
+  # https://github.com/canonical/cloud-init/blob/master/cloudinit/subp.py#L373
```

Comment:
This comment is amazing!!! :smile:

Reply:
![image](https://user-images.githubusercontent.com/21343441/119693043-eb88c100-be19-11eb-9852-95ff259e5210.png)

Reply:
Since you stole this from Dad, you get the same comment.  Does it make sense to space these out to allow thangs in between?  `10`, `20`...  
![iu-1](https://user-images.githubusercontent.com/3229435/119889858-1996ff80-bf05-11eb-8471-4a423a4819d7.jpeg)

Reply:
> Since you stole this from Dad, you get the same comment. Does it make sense to space these out to allow thangs in between? `10`, `20`...

I'll steal his reply too:  https://github.com/cisagov/cool-assessment-terraform/pull/118#discussion_r640925051